### ISSUE TITLE
Downloading without internet 

### ISSUE THREAD

**Issue by kanzaali** [samottaghi/adversarial-representation-active-learning#2]
Hi, 
Thank you so much for sharing your code. 

I am implementing your project. On the server, the internet is disabled for security reasons,  how can I resolve the downloading issue for this project on my personal desktop?

Downloading http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz

I have also tried by downloading the dataset on my personal computer and copy files on the server.

**Comment by alimottaghi** [comment]
Thank you so much for your interest. You can use the following zip file by extracting it into the `data` folder inside the root directory:
[data.zip](https://drive.google.com/file/d/1i45egBxfyTY_6_A0fT19HWHx2eeNx_v7/view?usp=sharing)

I hope this helps! Also, feel to reach out if you have any questions or concerns.

**Comment by kanzaali** [comment]
how to disable this line+ Downloading http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz

**Comment by kanzaali** [comment]
![Screenshot from 2020-03-16 13-23-29](https://user-images.githubusercontent.com/33715985/76723104-80f5e300-6789-11ea-8020-8b2b831f2904.png)

**Comment by alimottaghi** [comment]
Pytorch will not download the dataset again if it finds it in the right place. But you can disable downloading in `data.py` file lines 66-67, 90-91, and 122-123.

Please make sure that the data is located at your`data_root` folder. It is defined in the `config.py` file.

**Comment by kanzaali** [comment]
yes, you are right, I don't know why it cannot find the data set, though I have checked the directory again and again. 

Thank you so much for your detail response.
